### PR TITLE
Improve test

### PR DESCRIPTION
**Pull request by kiyonlin**
**Please provide enough information so that others can review your pull request:**

**Explain the *details* for making this change. What existing problem does the pull request solve?**

**Commit formatting** 

Use emojis on commit messages so it provides an easy way of identifying the purpose or intention of a commit. Check out the emoji cheatsheet here: https://gitmoji.carloscuesta.me/

Co-authored-by: Rene rene@gofiber.io

### CODE REVIEW

File: middleware/cors/cors.go [hunk 1]
```diff
@@ -81,12 +81,6 @@ func New(config ...Config) fiber.Handler {
 		cfg = config[0]
 
 		// Set default values
-		if cfg.Next == nil {
-			cfg.Next = ConfigDefault.Next
-		}
-		if cfg.AllowOrigins == "" {
```
Comment (ReneWerner87) [inline middleware/cors/cors.go:87]: why you remove that ?
default allowOrigins is "*" and should be used, when nothing is added in the custom user config

File: middleware/cors/cors_test.go [hunk 2]
```diff
@@ -41,7 +42,13 @@ func Test_CORS_Wildcard(t *testing.T) {
 	handler := app.Handler()
 
 	// OPTIONS (preflight) response headers when AllowOrigins is *
-	app.Use(New(Config{AllowOrigins: "*", AllowCredentials: true, MaxAge: 3600}))
+	app.Use(New(Config{
```
Comment (ReneWerner87) [inline middleware/cors/cors_test.go:45]: can you add an test with the default configuration ? and check then the expected headers/behavior

File: middleware/cors/cors.go (hunk 1 above)
Comment (kiyonlin) [reply middleware/cors/cors.go:87]: Yeah you are right. I made a mistake :smile:

File: middleware/cors/cors_test.go [hunk 3]
```diff
@@ -41,7 +42,13 @@ func Test_CORS_Wildcard(t *testing.T) {
...
+		AllowOrigins:     "*",
```
Comment (ReneWerner87) [inline middleware/cors/cors_test.go:46]: can you please add another test, where you leave an empty configuration

**Review by ReneWerner87 — APPROVED** [review]

File: middleware/cors/cors_test.go (hunk 3 above)
Comment (ReneWerner87) [reply middleware/cors/cors_test.go:57]: thx for the additional test case
